app/controllers/monthly_reports_controller.rb

```diff
@@ -16,7 +16,7 @@ def show
   end
 
   def new
-    target_month = params[:target_month] || Date.current.last_month.beginning_of_month
+    target_month = params[:target_month] || Date.current.last_month.since(5.days).beginning_of_month
```

Comment:
ともさんのPRでもこのロジックがかぶっちゃう問題あったので、

User#report_registrable_to
```
  def report_registrable_to
    Time.current.last_month.since(5.days)
  end
```

- このメソッドをpublicにする
- `beginning_of_month`までメソッドに含めちゃう

とかやったらいい感じに集約できないでしょうか？